### PR TITLE
Issue #1106 Fix scroll offset bug in search results

### PR DESCRIPTION
**Pull request by niko-pg**
… the listener config.

### CODE REVIEW

**Comment by niko-pg** [comment]
If ever you're curious about the Animated api: https://software-mansion.github.io/react-native-reanimated/index.html

**Comment by ian-lin-PG** [comment]
`onPress` event is not triggering consistently for Android devices. I can only tell an event is triggered properly when the item turns opaque which doesn't happen most of the time. The first item that successfully open will have this error message:
image
I have occasions have the service page open much later after I furiously tapped on the service item. When pressing the back button it does returns to correct position within the list.

**Comment by niko-pg** [comment]
@tomy-pg @ian-lin-PG I've basically hit a rabbit hole while fixing that I just decided to move the setting of the offset in `onScrollEndDrag`. In most of my testing, it works. But I'm worried that I missed some edge cases. Let me know if there are still issues.

File: src/components/search/search_results_component.tsx [hunk 1]
```diff
@@ -87,18 +86,18 @@ const renderComponentWithResults = (props: Props): JSX.Element => {
         }
     }, [props.searchOffset]);
 
-    const onScroll = (e: any): void => {
+    const onScrollEnd = (e: any) => {
+        console.log(e.nativeEvent.contentOffset.y)
```
Comment (ian-lin-PG) [inline src/components/search/search_results_component.tsx:90]: Noticed these logs in the output, otherwise looks good!

**Pull request closed without merging**